doc/commands/do.8.rst

```diff
@@ -0,0 +1,140 @@
+..
+    Copyright Contributors to the libdnf project.
```

Comment:
As this is a new file, please replace `libdnf` with `DNF5` (in several locations).

Reply:
Thanks. I like the idea of using **DNF5** instead of **libdnf** in the new files.

That reminds me, we once discussed using **SPDX** headers.